reporter/build.gradle.kts

```diff
@@ -23,6 +23,7 @@ val apachePoiSchemasVersion: String by project
 val cyclonedxCoreJavaVersion: String by project
 val flexmarkVersion: String by project
 val hamcrestCoreVersion: String by project
+val jacksonVersion: String by project
 val kotlinxHtmlVersion: String by project
 val simpleExcelVersion: String by project
```

Comment:
commit-message: In case you have a link / url which explains why `ObjectIdInfo` is not working for YAML it'd be nice to have it added to the commit message. 

Could you mention if there was an alternative to `ObjectIdInfo` which would work with YAML as well?

Reply:
I have added YAML serialization again and removed that line from the commit message.